src/main/java/com/bootcamp/demo/model/PaymentDetailsDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.bootcamp.demo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentDetailsDTO {
```

Comment:
Good ideea! DTO's are object that carries data between processes in order to reduce the number of methods calls. 
Just a FYI: DTO's can be used successfully for REST services as well, but usually in a REST API we can define a request object as part of the model (in this case a PaymentsRequest) especially since this is part of the API interface and an object that holds "a payment" data makes sense as a standalone object and not only a DTO.